assets/js/frontend/give-donations.js

```diff
@@ -992,6 +1089,20 @@ jQuery(function ($) {
 		$('a.give_terms_links', $fieldset_wrapper).toggle();
 		return false;
 	});
+
+	/**
+	 * Prevent level jump which happen due to same id.
+	 * @see https://github.com/WordImpress/Give/issues/2292
+	 */
+	$('label[for^="give-radio-level"]').on('click', function (e) {
```

Comment:
@DevinWalker this is quick fix for  https://github.com/WordImpress/Give/issues/2292. I think we need a logic to generate unique for donation form and also update logic (javascript mainly) which depends upon form or it's element ids.

Let me know what do you think.

Reply:
@ravinderk I agree. Each form embedded on a page should have a _unique_ ID. That means even if you're embedding the _same_ form on a page the twice that the two forms will still have _unique_ IDs. We're running into issues where people embedded the same for on a page which means duplicate IDs are in use within HTML (breaks semantic markup and validation) and also causes some complications with jumping between the forms for certain form elements.